maven-plugin/src/main/kotlin/AgentMojo.kt

```diff
@@ -80,11 +85,35 @@ abstract class AgentMojo : AbstractMojo() {
         }
     }
 
+    private fun getVersion(artifactory: String) = runCatching {
+        URL(
+            "$artifactory/api/search/latestVersion?" +
+                    "g=${config.artifactGroup}&a=${config.artifactId}-$presetName&v=+"
```

Comment:
Maybe is more readable in one line?
```suggestion
        URL("$artifactory/api/search/latestVersion?g=${config.artifactGroup}&a=${config.artifactId}-$presetName&v=+"
```

Reply:
I made a string in one line, without +. With URL it is too long.